doc-content/enterprise-only/employee-rostering/er-rotation-edit-proc.adoc

```diff
@@ -1,17 +1,32 @@
 [id='er-rotation-edit-proc']
+
 = Viewing and editing the rotation pattern
+The rotation pattern enables you to add, move, and delete shifts so you can manage your employee resources efficiently. It is defined by time buckets and seats.
+
+image::employee-rostering/rotation.png[]
```

Comment:
Need to remove the extra "B" with no circle background in the screenshot; otherwise good.

Reply:
Removed